src/meshapi/tests/test_maintenance_mode.py

```diff
@@ -0,0 +1,89 @@
+from django.contrib.auth.models import User
+from django.test import TestCase
+from django.test.client import Client
+from flags.state import disable_flag, enable_flag, flag_disabled, flag_enabled
+
+
+class TestMaintenanceMode(TestCase):
+    c = Client()
```

Comment:
nit: you can just use the built in `self.client` for this

Reply:
damn that would have been good to know all the way back last august.

Reply:
Yeah… I only noticed this a few weeks ago when looking at an example in the docs